grails-app/controllers/org/pih/warehouse/api/ApiController.groovy

```diff
@@ -113,6 +113,29 @@ class ApiController {
             ]
         ]
 
+        Map<Map> breadcrumbsConfig = [
+            "inbound": [
+                "label" : "react.stockMovement.inbound.label",
+                "defaultLabel": "Inbound stock movement",
```

Comment:
Currently, the breadcrumbs look like this: home > location > Stock Movement > Create inbound/outbbound/request Were the two last parts intended to be merged into one?
Unless the merge was in requirements, I think there might be a redirect from the "Stock Movement" breadcrumb into the stock movement list. 
Additionally, perhaps we could move this breadcrumbsConfig into config file.

Reply:
I will check that, I really had the impression that nothing in the breadcrumbs was redirecting to the list of stock movements

Reply:
But I feel like it would be a proper approach to it. @jmiranda what do you think about this?
We could have: 
`home > location > Inbound/outbound/request Stock Movements (with link into proper list) > create/edit > SM code/name`
or as it is now, but with 'Stock movement' part linking into proper list
`home > location > Stock Movements (with link into inbound/outbound/request list) > create/edit inbound/outbound/request > SM code/name`

Reply:
You are right, I will try that !

Reply:
Done for the list. I have tried for the edit but in the end when you click on the number of the stock movement, you are already redirected to the edit page so we had it twice ..

But everything is made now to insert actions other than create in the breadcrumbs, I have made some changes in the getAppContext for this